main.c

```diff
@@ -177,7 +177,7 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   mgmt_rtmsg_handler handler[] = {
       { ready,    OnReady },
       { hold,     OnHold },
-      { log,      OnLogLine },
+      { log_,      OnLogLine },
```

Comment:
While at it shall we decorate all those enums with an underscore -- like echo_ and ready_? Or all caps?

Reply:
Agreed, I'll add _ to the rest enum names for consistency.

Reply:
Looks good now. 
For other changes, you now better and @chipitsine has approved. Merging.